src/entrypoints/core.ts

```diff
@@ -32,6 +32,20 @@ declare global {
   }
 }
 
+// https://github.com/home-assistant/frontend/pull/7031
+const isSafari14 = /^((?!chrome|android).)*version\/14\.0.*safari/i.test(
```

Comment:
Why the guard for Chrome / Safari ?

Reply:
Chrome/android also has safari in the user string

Reply:
```
"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/85.0.4183.102 Safari/537.36"
```

Reply:
lol that string is so useless.